conf/icr_alma.config

```diff
@@ -0,0 +1,48 @@
+
+/*
+    * -------------------------------------------------
+    *  Nextflow nf-core config file for ICR alma HPC
+    * -------------------------------------------------
+    * Defines slurm process executor and singularity
+    * settings.
+    *
+    */
+params {
+    config_profile_description = "Nextflow nf-core profile for ICR alma HPC"
+    config_profile_contact = "Rachel Alcraft (@rachelicr)"
+}
+
+process {
+    queue="compute"
+    executor = "SLURM"
+    shell  = ['/bin/bash', '-euo', 'pipefail']
+    // memory errors which should be retried. otherwise error out
+    errorStrategy = { task.exitStatus in [143,137,104,134,139,140,247] ? 'retry' : 'finish' }
```

Comment:
You shouldn't need to define this here, both are already predefined in each pipeline. Particularly the error strategy one some pipelines may have extended this list, and you don't want to override those